topics/visualisation/tutorials/circos/tutorial.md

```diff
@@ -273,29 +274,39 @@ In this section we will reproduce this Circos plot step by step.
 As the first step to this Circos plot, let's configure the ideogram (set of chromosomes to draw). You can use one of the built-in genomes, or you can supply your own karyotype file.
 
 
+> ### {% icon hands_on %} Hands-on: Concatenate karyotype definitions and bands
+>
+> 1. {% tool [Concatenate datasets tail-to-head](toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_cat/0.1.0) %}:
+>     - {% icon param-file %} *"Concatenate Datasets"*: `hg18_karyotype.txt`
+>     - {% icon param-repeat %} *"Insert Dataset"*
+>         - In *"1. Dataset"*:
+>             - {% icon param-file %} *"Select"*: `hg18_karyotype_bands.tsv`
+>
+> 2. **Rename** {% icon galaxy-pencil%} the output as `hg18_karyotype_combined.txt`
+>
+>    {% include snippets/rename_dataset.md %}
+{: .hands_on}
```

Comment:
Ahh we had previously intentionally separated it. I think it makes sense to update the zenodo record here and just concatenate the two. @shiltemann that record is yours, could you concatenate the karyotype with the bands file? (just add to the karyotype file is probably best)

Reply:
I tried to use only the karyotype, but it didn't work for me.

Reply:
Yeah, I remember. My suggestion here to @shiltemann is that we take the karyotype + bands, and produce a single file (now just called hg18_karyotype.txt) so users don't have to merge as the first step (most of the karyotypes available from within circos are like this anyway)